packages/utils/src/types.ts

```diff
@@ -683,7 +683,7 @@ export type ObjectFieldTemplateProps<
   /** A string value containing the title for the object */
   title: string;
   /** A string value containing the description for the object */
-  description?: string;
+  description?: string | JSX.Element;
```

Comment:
`ReactNode` is the better choice for this
```suggestion
  description?: ReactNode;
```